math/vecops/CMakeLists.txt

```diff
@@ -31,7 +31,7 @@ endif()
 
 if(MSVC)
   target_compile_options(ROOTVecOps PRIVATE -O2 /fp:fast)
-  target_compile_definitions(ROOTVecOps PRIVATE _USE_MATH_DEFINES)
+  target_compile_definitions(ROOTVecOps PRIVATE _USE_MATH_DEFINES) # for M_PI in math.h
```

Comment:
Why not keeping the `#define _USE_MATH_DEFINES` inside the header?

Reply:
Well, it seems that this does not work and has no effect on `math.h`. Putting it in the header works (such as you suggested initially).